crates/common/consensus/src/execution_engine/mod.rs

```diff
@@ -166,4 +169,26 @@ impl ExecutionEngine {
             .await?
             .to_result()
     }
+
+    pub async fn engine_forkchoice_updatedv3(
+        &self,
+        forkchoice_state: ForkchoiceStateV1,
+        payload_attributes: PayloadAttributesV3,
```

Comment:
```suggestion
        payload_attributes: Option<PayloadAttributesV3>,
```

The spec says this is optional